tests/Yaapii.Atoms.Tests/Scalar/OrTests.cs

```diff
@@ -44,6 +43,69 @@ public void TrueOrGenScalar()
                     ).Value() == true
             );
         }
-    }
-}
 
+        [Fact]
+        public void TestFunc()
```

Comment:
Please make a name for the test which tells us what it tests

Reply:
Done